### PR TITLE
Update memory.md

### PR DESCRIPTION
**Pull request by JamesH65**
Try and be clearer on use of gpu_mem

### CODE REVIEW

**Comment by timg236** [comment]
Looks good to me. 

@6by9 Is there a sensible number for gpu_mem which we can add for a demanding (but plausible) MMAL/Camera configuration?

**Comment by JamesH65** [comment]
I wonder if a table or similar with the amounts required for each feature might be useful - people can them just add things up.

**Comment by 6by9** [comment]
On 1GB Pis I thought the default was 80MB as it reclaimed the 16MB hole that the ARM can't get to but the VPU can. 64+16 = 80.

80MB is sufficient for most video codecs stuff using VLC etc.
Video transcode will want 128MB to allow for the encode side as well as decode.
128MB is recommended for the camera, although higher resolutions of the HQ camera through PiCamera want 160MB.

**Comment by lurch** [comment]
This looks fine to me.
Are people still trying to use nonsensical `gpu_mem` values, despite the earlier warnings you added to this file?

File: configuration/config-txt/memory.md [hunk 1]
```diff
@@ -2,20 +2,24 @@
...
+Specifies how much memory, in megabytes, to reserve for the exclusive use of the GPU: the remaining memory is allocated to the ARM CPU for use by the OS. For Pis with less than 1GB of memory, the default is `64`; for Pis with 1GB or more of memory the default is `76`. 
+
+The memory allocated to the GPU is used for display, 3D, Codec and camera purposes as well as some basic firmware housekeeping. The maximums specified below assume you are using all these features. If you are not, then smaller values of gpu_mem can be used. 
+
+To ensure the best performance of Linux, you should set `gpu_mem` to the lowest possible value. If a particular graphics feature is not working correctly, try increasing the value of `gpu_mem`, being mindful of the recommended maximums shown below. Unlike GPU's found on x86 machines, where increasing memory can improve 3D performance, the architecture of the Videocore means **there is no performance advantage from specifying values larger than is necessary**, and in fact it can harm performance.
```
Comment (lurch) [inline configuration/config-txt/memory.md:9]: Capital 'C' on VideoCore?

**Pull request merged**